wled00/FX.h

```diff
@@ -977,4 +1008,4 @@ class WS2812FX {  // 96 bytes
 extern const char JSON_mode_names[];
 extern const char JSON_palette_names[];
 
-#endif
+#endif
```

Comment:
Might be worth checking your IDE settings to not automatically remote trailing end of line chars